content/en/containers/docker/_index.md

```diff
@@ -229,15 +229,22 @@ Additional examples are available on the [Container Discover Management][25] pag
 
 **Note**: When using containerd, it's possible to ignore containers by namespace using `DD_CONTAINERD_NAMESPACES` and `DD_CONTAINERD_EXCLUDE_NAMESPACES`. Both are a space-separated list of namespaces. When `DD_CONTAINERD_NAMESPACES` is set, the agent reports data for the containers that belong to a namespace present in the list. When `DD_CONTAINERD_EXCLUDE_NAMESPACES` is set, the agent reports data for all the containers except the ones that belong to a namespace of the list.
 
+### Autodiscovery
+
+| Env Variable                 | Description                                                                                                                                                                           |
+|------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DD_LISTENERS`               | Autodiscovery listeners to run.                                                                                                                                                       |
+| `DD_EXTRA_LISTENERS`         | Additional Autodiscovery listeners to run. They are added in addition to the variables defined in the `listeners` section of the `datadog.yaml` configuration file.                   |
+| `DD_CONFIG_PROVIDERS`        | The providers the Agent should call to collect checks configurations. The default provider is `docker`. The Docker provider handles templates embedded in container labels. |
```

Comment:
Should the description for `DD_CONFIG_PROVIDERS` in this Docker-specific doc match the description in the Kubernetes-specific docs? It's the same configuration option and the supported values are the same, but in Docker, only the `docker` value is applicable.

Reply:
let's keep what you have! i think it works very well!